src/i18n/index.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
```

Comment:
I would prefer not having compiled JS files in the repo (other than in the dist folder).

Reply:
Ah they were a mistake anyways I tested smth with the code from https://github.com/Drarig29/brackets-manager.js/issues/48#issuecomment-713010349